examples/cactus-example-supply-chain-backend/package.json

```diff
@@ -80,7 +85,11 @@
   },
   "devDependencies": {
     "@types/express": "5.0.1",
-    "@types/uuid": "10.0.0"
+    "@types/jest": "^29.5.14",
```

Comment:
Please pin the dependencies version (i.e. `29.5.14`). We use pinned versions to prevent supply chain attacks.

Reply:
This applies to all package.json affected by this PR